### PR TITLE
[transmission] initcontainer fix

### PR DESCRIPTION
**Pull request by jr0dd**
**Checklist** 
- [ x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x ] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x ] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Review by onedr0p — DISMISSED** [review]

File: charts/stable/transmission/values.yaml [hunk 1]
```diff
@@ -5,19 +5,21 @@
...
+#  command:
+#  - "/bin/sh"
+#  - "-c"
+#  - "curl -o- -sL https://github.com/johman10/flood-for-transmission/releases/download/latest/flood-for-transmission.tar.gz | tar xzf - -C /config"
+#  volumeMounts:
+#  - name: config
+#    mountPath: /config
+#  securityContext:
+#    runAsUser: 568
+#    runAsGroup: 568
```
Comment (onedr0p) [inline charts/stable/transmission/values.yaml:22]:
```suggestion
#initContainers:
# - name: custom-webui
#   image: curlimages/curl:7.76.1
#   command:
#   - "/bin/sh"
#   - "-c"
#   - "curl -o- -sL https://github.com/johman10/flood-for-transmission/releases/download/latest/flood-for-transmission.tar.gz | tar xzf - -C /config"
#   volumeMounts:
#   - name: config
#     mountPath: /config
#   securityContext:
#     runAsUser: 568
#     runAsGroup: 568
```

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: charts/stable/transmission/values.yaml (hunk 1 above)
Comment (onedr0p) [reply charts/stable/transmission/values.yaml:22]: This should fix the linting issues hopefully

Comment (jr0dd) [reply charts/stable/transmission/values.yaml]: > This should fix the linting issues hopefully

ah didn't even think about that after commenting out after testing

Comment (onedr0p) [reply charts/stable/transmission/values.yaml]: I think it's still messed up. You should uncomment this and then let vscode comment the lines for you.

**Comment by jr0dd** [comment]
@onedr0p alright i used VS Code to comment the lines. looks like it added 1 more space after #

File: charts/stable/transmission/README.md [hunk 2]
```diff
@@ -156,4 +156,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Join our [Discord](https://discord.gg/sTMX7Vh) community
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
+Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
```
Comment (onedr0p) [inline charts/stable/transmission/README.md:159]:
```suggestion
Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)

```

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
@all-contributors add @jr0dd for code

**Pull request merged**